common/reviews/api/api-extractor.api.ts

```diff
@@ -1,3 +1,4 @@
+// @public
```

Comment:
running generate brought in a new version of api-extractor, which created a bunch of warning messages. These aren't changing any of the API signatures, it's just appeasing api-extractor.